doc/api/net.md

```diff
@@ -722,25 +722,26 @@ that the
 [`socket.connect(options[, connectListener])`][`socket.connect(options)`]
 callback is a listener for the `'connect'` event.
 
-### `socket.destroy([exception])`
+### `socket.destroy([error])`
 <!-- YAML
 added: v0.1.90
 -->
 
-* `exception` {Object}
+* `error` {Object}
 * Returns: {net.Socket}
 
-Ensures that no more I/O activity happens on this socket. Only necessary in
-case of errors (parse error or so).
+Ensures that no more I/O activity happens on this socket. It effectively discards
+any buffered data and ends the socket.
```

Comment:
This is a bit misleading. Buffered data on the readable side of the socket is not discarded.

Reply:
Good spot.

Reply:
Removed.